PocketKit/Sources/PocketKit/PocketAppDelegate.swift

```diff
@@ -94,6 +94,7 @@ public class PocketAppDelegate: UIResponder, UIApplicationDelegate {
         do {
             let attempted = try legacyUserMigration.perform()
             if attempted {
+                Services.shared.braze.signedInUserDidBeginMigration()
```

Comment:
Would it be possible to add a `braze` (or similar name) property, similarly to what we do with `source`, `userDefaults` etc instead of directly calling the singleton here? This would make it easier if we switch to a full dependency injection pattern here.

Reply:
This is actually already the case. The `braze` property is of type `BrazeProtocol`

Reply:
Actually what I meant (sorry for the confusion) is to replace

`Services.shared.braze.signedInUserDidBeginMigration()`

with 

`brazeService.signedInUserDidBeginMigration()`

`brazeService` would be an additional property in `PocketAppDelegate`, initialized from `Services`, similarly to the other existing properties.